purchase_order_analytic_search/README.rst

```diff
@@ -0,0 +1,90 @@
+==============================
+Purchase Order Analytic Search
+==============================
+
```

Comment:
I think you can remove this file.
Once it is merged, it will be created auto. Just make sure that "readme" folder is good.